frodo.js

```diff
@@ -2,93 +2,99 @@
  * Extras Plugin
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?